### PR TITLE
React update

### PR DESCRIPTION
**Pull request by laves**

### CODE REVIEW

File: sdk/react/src/use_picovoice.ts [hunk 1]
```diff
@@ -68,38 +60,31 @@ export function usePicovoice(
...
+  const init = useCallback(
+    async (
+      accessKey: string,
+      keyword: PorcupineKeyword,
+      porcupineModel: PorcupineModel,
+      context: RhinoContext,
+      rhinoModel: RhinoModel,
+      options: PicovoiceOptions = {}
+    ): Promise<void> => {},
+    [wakeWordCallback, inferenceCallback]
```
Comment (ksyeo1010) [inline sdk/react/src/use_picovoice.ts:80]: should we add all of them as deps?

**Review by ksyeo1010 — APPROVED** [review]

File: sdk/react/src/use_picovoice.ts (hunk 1 above)
Comment (ErisMik) [reply sdk/react/src/use_picovoice.ts:80]: They aren't deps, but arguments into the callback so it's okay.

**Review by ErisMik — APPROVED** [review]

**Review by ksyeo1010 — APPROVED** [review]

File: sdk/react/src/use_picovoice.ts [hunk 2]
```diff
@@ -9,230 +9,178 @@
...
+    keyword: PorcupineKeyword,
+    porcupineModel: PorcupineModel,
+    context: RhinoContext,
+    rhinoModel: RhinoModel,
+    options?: PicovoiceOptions
+  ) => Promise<void>;
+  start: () => Promise<void>;
+  stop: () => Promise<void>;
+  release: () => Promise<void>;
 } {
```
Comment (ErisMik) [inline sdk/react/src/use_picovoice.ts:45]: tslint and prettier don't agree on this it's very annoying
```suggestion
  } {
```

File: sdk/react/src/use_picovoice.ts [hunk 3]
```diff
@@ -9,230 +9,178 @@
...
+        picovoiceRef.current.terminate();
+        picovoiceRef.current = null;
+        setIsLoaded(false);
+      }
+    } catch (e: any) {
+      setError(e.toString());
+    }
+  }, [stop]);
+
+  useEffect(() => {
```
Comment (ErisMik) [inline sdk/react/src/use_picovoice.ts:168]: Try this to pass the linter:

```
useEffect(() => (): void => {
    release();
}, []);
```

**Pull request merged**